api/create_hypertable.md

```diff
@@ -17,7 +17,7 @@ still work on the resulting hypertable.
 |Name|Type|Description|
 |---|---|---|
 | `relation` | REGCLASS | Identifier of table to convert to hypertable. |
-| `time_column_name` | REGCLASS | Name of the column containing time values as well as the primary column to partition by. |
+| `time_column` | REGCLASS | Name of the column containing time values as well as the primary column to partition by. |
```

Comment:
This should not be changed, the parameter name is `time_column_name`. 

The datatype is wrong however, it's a `NAME` not a `REGCLASS`.

Reply:
Same with partitioning column below- should also technically be a name and not a regclass

Reply:
And partitioning_func / time_partitioning_func -> `REGPROC`
associated_schema_name -> `NAME`